src/Executables/Benchmark/Benchmark.cpp

```diff
@@ -88,11 +86,12 @@ void bench_all_gradient(benchmark::State& state) {  // NOLINT
   constexpr const size_t Dim = 3;
   const Mesh<Dim> mesh{pts_1d, Spectral::Basis::Legendre,
                        Spectral::Quadrature::GaussLobatto};
-  CoordinateMaps::Affine map1d(-1.0, 1.0, -1.0, 1.0);
-  using Map3d = CoordinateMaps::ProductOf3Maps<CoordinateMaps::Affine,
-                                               CoordinateMaps::Affine,
-                                               CoordinateMaps::Affine>;
-  CoordinateMap<Frame::Logical, Frame::Grid, Map3d> map(
+  domain::CoordinateMaps::Affine map1d(-1.0, 1.0, -1.0, 1.0);
```

Comment:
Why wasn't this caught in CI?

Reply:
I think while switch to GitHub Actions the benchmark build was lost, unfortunately...